pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+```
+{
+      "project_name": "example",
+      "project_url": "https://github.com/your-github/example",
+      "author": "Your Name",
+      "description": "a quick description to summarize functionality",
+      "discussion_url": "https://llllllll.co/t/example/000000",
+      "tags": ["example", "example", "example"],
+      "origin": "(if applicable) lines"
```

Comment:
The `origin` property is a leftover from the lines forum scraper and not really needed. It is probably just as well that we remove it.